index.html

```diff
@@ -201,35 +161,50 @@
   // always starting from a fresh copy of the data, and we can count on the result to be consistent
   // given the same values in options.
   var options = {
+    candidate: false,
     sorted: false,
     filtered: false,
     page: 0
   };
 
   function updateTable() {
     // Make a copy of the data
-    var data = schools.slice();
-
-    // Sort schools by enrollment
-    if (options.sorted) {
-      if (options.sorted === 'asc') {
-        data.sort(function (a, b) { return a.enrollment - b.enrollment; });
-        // This anonymous function is responsible for comparing two values a and b. If it returns a positive
-        // number, we're saying that a should come after b in the array. A negative number means that b
-        // should come after a. A zero means that the two are equal. This function is using the 'enrollment'
-        // property of each observation of data for this comparison. You'll need to change 'enrollment' to
-        // whatever property you want to sort your data on.
-        //
-        // Also, note that we're sorting the data in place, changing, or mutating, the 'data' variable. See
-        // the note in the README about how this compares to the filter() and slice() operations that we'll
-        // use for the next two data transforms.
-      } else if (options.sorted === 'desc') {
-        data.sort(function (a, b) { return b.enrollment - a.enrollment; });
-        // Note that this is just the opposite of the above.
+    var data = dem_delegates.slice();
+
+    // Sort election by delegate count and candidate
+    // Is there a way to structure the code such that I don't need to add the "&& options.candidate === 'clinton'"?
+    // In something like SAS or R we could write the return statement like {return a.[options.candidate] - b.[options.candidate]}
+    // in order to filter on what we've already set as the filter criteria somewhere else.
+    if (options.sorted && options.candidate) {
+
+      // Ascending sort
+      if (options.sorted === 'asc' && options.candidate === 'clinton') {
```

Comment:
This absolutely works. Would love to see you rework it as we discussed in class to use the `object[key]` syntax, as opposed to the dot syntax (`object.key`) used here, in order to simplify the logic.

Also, you're maybe starting to see how it can be useful to standardize capitalization. I generally advocate for using Javascript naming conventions (camelCase) for data column names, just so you don't have to switch back and forth. That being said, snake_case is also acceptable and commonly used; Javascript coding styles do, however, avoid the use of variables that start with capital letters unless they are all-caps CONSTANTS, unchanging values like `TOTAL_DELEGATES`, or a [constructor function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new).